src/providers/nextbus.js

```diff
@@ -43,14 +43,14 @@ function makeVehicle(nextbusObject) {
     var secsSinceReportInt = parseInt(secsSinceReport, 10);
 
     return {
-      rid: routeTag,
-      vid: id,
-      lat: latFloat,
-      lon: lonFloat,
+      routeId: routeTag,
+      vehicleId: id,
+      latitude: latFloat,
+      longitude: lonFloat,
       heading: isNaN(headingInt) ? null : headingInt,
-      did: dirTag,
+      directionTag: dirTag,
```

Comment:
Could this be directionId to be more consistent with terminology used in opentransit-metrics?

Reply:
I was thinking a similar thing. The response value does not fit with the `directionId` value but we could parse it and then store it so that it's similar. Here's an example response:
<img width="253" alt="image" src="https://user-images.githubusercontent.com/24400820/154396637-d18b7eac-54f0-42ee-b425-be03649e7438.png">
It could be worth looking into how to parse that field. What do you think?

Reply:
Yeah, actually the Nextbus dirTag is currently ignored by opentransit-metrics because it is not reliable. I think the dirTag is set by the operator but sometimes they forget to change the direction at the end of a route.  It is more reliable to determine the direction of the vehicle based on the progression of latitude and longitude. We could potentially stop storing dirTag to save space in S3, or keep storing it in case we find a use for it in the future. If we keep storing it, using directionTag or dirTag is fine with me.